### ISSUE TITLE
brew search meroxa does not show older versions

### ISSUE THREAD

**Issue by janelletavares** [meroxa/cli#738]
**Describe the bug**  

From a high level, describe the problem you ran into.

**Machine (please complete the following information):**  

 - CLI Version [e.g. 22]

**To Reproduce**  

What can we do to reproduce your bug?

Please provide:
- Relevent CLI commands
- Relevent environment details

**Expected behavior**  
What did you expect to happen? 

**Screenshots**  
If applicable, add screenshots to help explain your problem. (Be sure to scrub any sensitive information.)

**Additional context**

**Comment by raulb** [comment]
I don't think this natively supported by GoReleaser so we'd need to advice users who need to do that differently https://github.com/goreleaser/goreleaser/issues/1058

**Comment by raulb** [comment]
@janelletavares this might be relevant to you https://blog.sandipb.net/2021/09/02/installing-a-specific-version-of-a-homebrew-formula/. Would you like to try that one out and see if that worked.

**Comment by janelletavares** [comment]
@raulb thank you for the links. my application was on github, but have since gone another way. i have not been using brew on linux. @justmisosoup Sara says the experience around using older versions isn't as important as a smooth upgrade, so it sounds like the priority on this is low - will-not-do at the moment.
